votes/2021-09-29-0.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "Moving an inactive collaborator to emeritus",
+  "refs": ["https://github.com/nodejs/node/pull/40115#issuecomment-930656531"],
+  "votes": {
+    "Trott": 1,
+    "ronag": 1,
+    "mcollina": 1,
+    "apapirovski": 1,
+    "aduh95": 1,
+    "cjihrig": 1,
+    "targos": 1,
+    "BridgeAR": 1,
+    "fhinkel": 1,
+    "jasnell": 1,
+    "mhdawson": 1,
+    "gireeshpunathil": 0
```

Comment:
checking at the referenced issue comment, I see that I have abstained from voting for the said topic. just want to understand, is that what `0` means here?

Reply:
ah, forget the comment, I see it is described below. thanks!